YellowNotes.Api/appsettings.json

```diff
@@ -6,5 +6,8 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "Jwt": {
+    "SecretKey": "secret here"
```

Comment:
This can be removed because ultimately this key will be placed in the key vault.